trunk/includes/pardot-plugin-class.php

```diff
@@ -270,16 +282,47 @@ function wp_ajax_get_pardot_dynamicContent_shortcode_select_html() {
 		die();
 	}
 
+
+	/**
+	 * AJAX function used to delete HTML cache of dynamic content & forms
+	 *
+	 * @since 1.5.7
+	 */
+	public function wp_ajax_delete_asset_html_transient()
```

Comment:
This is the new AJAX function I made that is called when we select a form or dynamic content in the WordPress plugin.  It will delete the existing cached HTML for the dynamic content or form.  It will pull and cache the latest version upon the next load of the asset.